src/bramble/client/main.js

```diff
@@ -27,10 +27,11 @@ define([
     // Change this to filer vs. filer.min if you need to debug Filer
     "thirdparty/filer/dist/filer.min",
     "bramble/ChannelUtils",
-    "bramble/thirdparty/EventEmitter/EventEmitter.min",
+    "bramble/thirdparty/EventEmitter/EventEmitter",
```

Comment:
Still not dealt with.  Please don't ask for a review while you have outstanding issues from a previous review, it wastes everyone's time.